docker/wc-admin-php-test-suite/docker-compose.yml

```diff
@@ -2,15 +2,19 @@ version: '3'
 
 services:
     phpunit:
-        build: '.'
-        image: wc-admin-php-test-suite-phpunit:1.2.1
+        build:
+            context: '.'
+            args:
+                - PHP_VERSION=${PHP_VERSION:-7}
+        image: wc-admin-php-${PHP_VERSION:-7}-test-suite-phpunit:1.2.2
         volumes:
-            - 'test-suite:/tmp'
+            - 'test-suite-php-${PHP_VERSION:-7}:/tmp'
             - '../../:/app'
         environment:
             - WC_CORE_DIR=/tmp/wordpress/wp-content/plugins/woocommerce
-            - WC_VERSION=${WC_VERSION:-4.8.0}
```

Comment:
These were intended to be our minimum supported versions. The idea is that we use our min version as default so that less effort is needed for local development. This is because things that work in the min version should also work in the newer versions due to backwards compatibility.

I think I need to document this better 😄

Reply:
Yeah, I changed it because PHP8 is not support till WP 5.6 and WC 5.1 (I think). I changed this back and in the `entrypoint.sh` I added some logic that sets the version to 5.6 and 5.1 automatically if we are running PHP 8 and it doesn't have higher versions set.